src/main/java/jenkins/metrics/impl/JenkinsMetricProviderImpl.java

```diff
@@ -487,6 +488,12 @@ protected Integer loadValue() {
                         return count;
                     }
                 }),
+                metric(name("jenkins", "plugins", "withWarnings"), new CachedGauge<Integer>(5, TimeUnit.MINUTES) {
+                    @Override
+                    protected Integer loadValue() {
+                        return ExtensionList.lookupSingleton(UpdateSiteWarningsMonitor.class).getActivePluginWarningsByPlugin().size();
```

Comment:
`UpdateSiteWarningMonitor` usage is restricted.

Reply:
Oh it is? Didn't see that. I'll check

Reply:
Hm you're right... Then I have no idea how to get that information. Well if you also don't know a way you can probably close the PR